### PR TITLE
New pvp clients tag

### PR DESCRIPTION
**Pull request by AsinusGrandus**
adds a new tag about pvp-clients

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -72,5 +72,13 @@
             "categories-2"
         ],
         "url": "tags/tournament/modes2.md"
+    }, {
+        "identifier": "pvp-clients",
+        "aliases": [
+            "lunar",
+            "badlion",
+            "optifine"
```
Comment (akiraveliara) [inline index.json]: optifine is not a pvp client; thats its own separate issue

File: tags/pvp-clients [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+One of those restrictions is the "no cape" rule, which is found in the section "Servers and Hosting" in the EULA:
+"Therefore, if you comply with and follow the Brand and Asset Usage Guidelines and Naming Guidelines above, YOU MAY sell cosmetic entitlements, except for "Capes," which we define as anything that attempts to visually emulate the feature of a Minecraft player Cape."
+From OptiFine straight up selling capes to Lunar, Badlion et al selling Wings, which looks a lot and has the same features as a Minecraft player Cape, here's undeniable proof they violate the Terms of Service. 
+And Mojang hasn't really expressed their agreement, either - in fact, there are several unofficial messages from Mojang Studios members stating those softwares do violate the EULA and ToS.
+Mojang is not in a position to take, say, OptiFine down. Even though they would be legally right, they couldn't take the community backlash, even though OptiFine doesn't actually optimize any more in the first place.
+A much better and more in debt explenation can be found at: https://gist.github.com/ExaInsanity/04519cf06ea122a4a5a7d66f1a930807#file-pvp-clients-md
```
Comment (akiraveliara) [inline tags/pvp-clients]: in-depth

**Comment by akiraveliara** [comment]
@Zefyro what do you think, other than the formalities?

File: index.json (hunk 1 above)
Comment (AsinusGrandus) [reply index.json]: Updated it, removed optifine from aliases

File: tags/pvp-clients (hunk 2 above)
Comment (akiraveliara) [reply tags/pvp-clients]: resolved

**Pull request merged**